modules/ephemeral-storage-csi-inline-overview.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/ephemeral-storage-csi-inline.adoc
+
+[id="ephemeral-storage-csi-inline-overview_{context}"]
+= Overview of CSI inline volumes
+
+Traditionally, volumes that are backed by Container Storage Interface (CSI) drivers can only be used with a PersistentVolume and PersistentVolumeClaim object combination.
+
+This feature allows you to specify CSI volumes directly in the Pod specification, rather than in a PersistentVolume. Inline volumes are ephemeral and do not persist across Pod restarts.
+
+== Support limitations
+
+By default, {product-title} supports CSI inline volumes with these limitations:
+
+* Support is only available for CSI drivers. In-tree and FlexVolumes are not supported.
+* {product-title} does not include any CSI drivers. Use the CSI drivers provided by
+link:https://kubernetes-csi.github.io/docs/drivers.html[community or storage vendors]. Follow the installation instructions provided by the CSI driver.
+* CSI drivers might not have implemented the inline volume functionality. For details, see the CSI driver documentation.
+* {product-title} {product-version} supports version 1.1.0 of the
+link:https://github.com/container-storage-interface/spec[CSI specification].
```

Comment:
The csi version should be 1.2.0,  I don't know if this is necessary here.

Reply:
> The csi version should be 1.2.0, I don't know if this is necessary here.

Good catch. Didn't realize it had been updated. I agree, probably not necessary, so I've deleted this bullet.